pip_faster.py

```diff
@@ -133,22 +133,25 @@ def find_requirement(self, req, upgrade):
             # if the version is pinned-down by a ==
             # first try to use any installed package that satisfies the req
             if req.satisfied_by:
-                logger.debug('Faster! pinned requirement already installed.')
+                logger.notify('Faster! pinned requirement already installed.')
```

Comment:
It's a bit of branding; this is a fast-path that's new in pip-faster.
Otherwise, if you accidentally use pip-slower the output looks just the same, but slower, hard to notice, especially under a CI situation.